doc/check-options.md

```diff
@@ -205,6 +205,7 @@ All checks allow these global options:
 | ----------------------------------------------------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `ignoreUnicode`                                             | `true`  | Do not check the color contrast of Unicode characters                                                                                                                                        |
 | `ignoreLength`                                              | `false` | Do not check the color contrast of short text content                                                                                                                                        |
+| `ignorePseudo`                                              | `false` | Do not check the color contrast of elements with pseudo-element styles                                                                                                                       |
```

Comment:
Since we don't really check them for contrast in the first place.

```suggestion
| `ignorePseudo`                                              | `false` | Do not mark pseudo elements as Needs Review                                                                                                                        |
```